android/src/main/java/com/mrousavy/camera/CameraViewModule.kt

```diff
@@ -283,6 +283,7 @@ class CameraViewModule(reactContext: ReactApplicationContext) : ReactContextBase
               format.putString("autoFocusSystem", "none") // TODO: Revisit getAvailableCameraDevices (autoFocusSystem) (CameraCharacteristics.CONTROL_AF_AVAILABLE_MODES or CameraCharacteristics.LENS_INFO_FOCUS_DISTANCE_CALIBRATION)
               format.putArray("videoStabilizationModes", videoStabilizationModes)
               format.putString("pixelFormat", pixelFormat)
+              format.putArray("supportedDepthDataFormats", Arguments.createArray())
```

Comment:
What this does for Android?

Reply:
Nothing, it's an empty array meaning there are no supported depth data formats on Android. I honestly don't know if Android cameras have depth sensors?